### ISSUE TITLE
Deployment fails with ERROR Rosco is enabled, and no connection to rosco could be established

### ISSUE THREAD

**Issue by mickengland** [spinnaker/halyard#796]
Trying to deploy Spinnaker in Kuberenetes with latest version of  Halyard on Ubuntu 14.0.4 and getting this error:

```
Problems in Global:
! ERROR Rosco is enabled, and no connection to rosco could be
  established: retrofit.RetrofitError: com.google.gson.JsonSyntaxException:
  java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1
  column 1 path $

- Failed to deploy Spinnaker.
```
Inside kubernetes all the pods show as being up and ready.

hal deploy connect works and I get an initial Hello screen but nothing else and if I refresh the browser I get 127.0.0.1 refused to connect.

 ```
$ hal deploy connect
+ Get current deployment
  Success
+ Connect to Spinnaker deployment.
  Success
Forwarding from 127.0.0.1:8084 -> 8084
Forwarding from 127.0.0.1:9000 -> 9000
```

Kubernetes is deployed in Rancher on AWS v1.8.3-rancher3 with RBAC enabled.

```
$ hal -v
0.39.0-20180103211607

$ uname -a
Linux vagrant-ubuntu-trusty-64 3.13.0-119-generic #166-Ubuntu SMP Wed May 3 12:18:55 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux

$ kubectl version
Client Version: version.Info{Major:"1", Minor:"9", GitVersion:"v1.9.0", GitCommit:"925c127ec6b946659ad0fd596fa959be43f0cc05", GitTreeState:"clean", BuildDate:"2017-12-15T21:07:38Z", GoVersion:"go1.9.2", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"8+", GitVersion:"v1.8.3-rancher3", GitCommit:"772c4c54e1f4ae7fc6f63a8e1ecd9fe616268e16", GitTreeState:"clean", BuildDate:"2017-11-27T19:51:43Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}

```
Any advice on this much appreciated.

**Comment by mickengland** [comment]
Still getting this issue with a clean deploy. Spinnaker UI now comes up in a browser but just spins when trying to access applications or projects.

**Comment by mickengland** [comment]
This seems to be fixed after upgrading to 1.5.4.
